scalameta/parsers/shared/src/main/scala/scala/meta/internal/parsers/ScalametaParser.scala

```diff
@@ -3828,7 +3832,7 @@ class ScalametaParser(input: Input)(implicit dialect: Dialect) { parser =>
   def primaryCtor(owner: TemplateOwner): Ctor.Primary = autoPos {
     if (owner.isClass || (owner.isTrait && dialect.allowTraitParameters) || owner.isEnum) {
       val mods = listBy[Mod] { buf =>
-        annotsBuf(buf, skipNewLines = false, allowArgss = false)
+        annotsBuf(buf, skipNewLines = false, allowArgss = false, insidePrimaryCtorAnnot = true)
```

Comment:
@tgodzik btw, what is `allowArgss = false` here? if i read the code that you shared correctly, at least dotty allows multiple, not sure about scala2.